docs/src/content/docs/components/magrepack.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: magrepack
+---
+
+"Magrepack" provides players with the ability to repack magazines. It was written by Skamdrept and Badger 2-3, and originally published under the name "SESOF MagRepack."
+
+# Usage
+Open the inventory and drag "Magazine 1" onto "Magazine 2" to transfer rounds to the latter. If "Magazine 1" is emptied in the process it will be dropped on the ground.
```

Comment:
```suggestion
Open the inventory and drag "Magazine 1" onto "Magazine 2" to transfer rounds to the latter. If "Magazine 1" is emptied in the process, it will be dropped on the ground.
```